### DISCUSSION TITLE
[@types/react] missing "children" in FC (v18)

### DISCUSSION THREAD

**Discussion by Torsten85** [DefinitelyTyped/DefinitelyTyped#59760] [Issues with a @types package] [answered]
Hello,

in Version 18, this line...
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L511
... no longer wraps the provided props with `PropsWithChildren`.

Before
```ts
interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
```

Now
```ts
interface FunctionComponent<P = {}> {
        (props: P, context?: any): ReactElement<any, any> | null;
```

Was this intentional or is this a bug?

Thanks

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
This change was intended. Check out https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210 for a full list of intended, breaking changes and their rationale. The PR includes links to a codemod that helps migrating to `@types/react@18`

**Reply by Torsten85** [reply]
Thank you
